app.js

```diff
@@ -0,0 +1,106 @@
+//Global Templates and variables
+var preferences = [];
+
+var indexId = 0;
+
+var questionTemplate = '<div>'+
+						'<label class="js-questionLabel"></label>'+
+		 					'<select class="js-choice" id="">' +
+								'<option value="Yes"> Yes </option>' +
+								'<option value="No"> No </option>' +
+							'</select>' +
+						'</div>';
+
+var drinksTemplate = '<div><h3></h3></div>';
+
+
+//Questions
+var questions = function(questionInput){
+	this.question = questionInput;
+}
+
+questions.prototype.render = function(){
+	var questionToRender = $(questionTemplate);
+	questionToRender.find('.js-questionLabel').text((this.question));
+	questionToRender.find('select').attr('id', indexId);
+	indexId += 1;
+	return questionToRender;
+}
+
+function renderQuestionToDom(){
+	$('.js-form').append(new questions('Do ye like yer drinks strong?').render());
+	$('.js-form').append(new questions('Do ye like it with a salty tang?').render());
+	$('.js-form').append(new questions('Are ye a lubber who likes it bitter?').render());
+	$('.js-form').append(new questions('Would ye like a bit of sweetness with yer poison?').render());
+	$('.js-form').append(new questions('Are ye one for a fruity finish?').render());
+	$('.js-form').append('<input type="submit"></input>');
+}
+
+//Ingredients
+var ingredients = function(ingredientInput){
```

Comment:
Capitalizing the var for the constructor function is in keeping with good convention.